### PR TITLE
Styling bookstore

### PR DESCRIPTION
**Pull request by iambenkis**
In this step, I used Redux store to display books. You also added some UI elements to components and make them dispatch actions.

## Checklist 

- [ ] State is  immutable in reducers
- [ ] Use of ```Provider```
- [ ] Proper use of ```useDispatch```
- [ ] Proper use of ```useDispatch```
- [ ] Design match Zeplin's design.

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @iambenkis, kindly re-confirm the milestone you submitted this pull request for

**Comment by iambenkis** [comment]
The styling , the last milestone.
@prolajumokeoni

**Comment by prolajumokeoni** [comment]
>

noted, thanks

File: src/redux/books/books.js [hunk 1]
```diff
@@ -8,27 +14,56 @@ export const addBook = (book) => ({
...
+const initialState = [
+//   id: 1,
+//   title: 'Benjamin biography',
+//   author: 'Benkis',
+// },
+// {
+//   id: 2,
+//   title: 'L ame du monde',
+//   author: 'Frederic francois',
+// },
```
Comment (prolajumokeoni) [inline src/redux/books/books.js:31]: ### OPTIONAL

- [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code 😉

File: src/redux/books/books.js [hunk 2]
```diff
@@ -1,5 +1,11 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+// import store from "../configureStore";
```
Comment (prolajumokeoni) [inline src/redux/books/books.js:2]: ### OPTIONAL
 - [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code 😉

**Review by prolajumokeoni — APPROVED** [review]
Hi @iambenkis, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- See optional comments under review

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**